scripts/bump-version

```diff
@@ -6,7 +6,7 @@ from pathlib import Path
 from lib.charts import bump
 
 parser = argparse.ArgumentParser(description="Bump versions in given helm charts")
-parser.add_argument("charts", metavar="chart", type=Path, nargs='+', help="The path to the Chart.yaml file, or the directory containing it", required=True)
+parser.add_argument("charts", metavar="chart", type=Path, nargs='+', help="The path to the Chart.yaml file, or the directory containing it")
```

Comment:
hrm.. maybe we can fix this before merging.
(.venv) [whayutin@thinkdoe pelorus]$ ./scripts/bump-version -help
Traceback (most recent call last):
  File "/home/whayutin/OPENSHIFT/git/KONVEYOR/PELORUS/upstream/pelorus/./scripts/bump-version", line 9, in <module>
    parser.add_argument("charts", metavar="chart", type=Path, nargs='+', help="The path to the Chart.yaml file, or the directory containing it", required=True)
  File "/usr/lib64/python3.10/argparse.py", line 1405, in add_argument
    kwargs = self._get_positional_kwargs(*args, **kwargs)
  File "/usr/lib64/python3.10/argparse.py", line 1521, in _get_positional_kwargs
    raise TypeError(msg)

Reply:
actually I fixed this error by this line :)